torch/fx/experimental/Partitioner.py

```diff
@@ -0,0 +1,151 @@
+from torch.fx import Graph, GraphModule
+from torch.fx import Node
+from typing import Dict, List, Tuple
+"""a graph module may not fit into one backend.
+Partitioner class helps cut one graph into subgraphs (partitions),
+so that each partition could fit into different backends
+"""
+class Partition:
+    def __init__(self, partition_id: int) -> None:
+        self.parents: List[Partition] = []
+        self.children: List[Parttion] = []
+        """
+           One partion only has to fit into one device
+           'backend type' is the type of that device
+        """
+        self.partition_id = partition_id
+        """
+           self.node_list contains graph nodes that will be deployed in this partition
```

Comment:
Rather than have this information in a docstring, what about making a custom data structure that has each field named, or at least a named_tuple if it's used elsewhere in the codebase.